server/game/core/event/EventWindow.js

```diff
@@ -93,10 +102,11 @@ class EventWindow extends BaseStepWithPipeline {
     }
 
     executeHandler() {
+        this.triggeredAbilityWindow.addTriggeringEvents(this.events);
```

Comment:
Could we save a step and just pass the events in at `emitEvents()`?

Reply:
Yes.